schemas/logs/fluentd/windows_eventlog.yml

```diff
@@ -0,0 +1,136 @@
+schema: Fluentd.WindowsEventlog
+parser:
+  native:
+    name: Fluentd.WindowsEventlog
```

Comment:
I think `native` is reserved for log types that have a go parser on the backend.